storage/innobase/include/trx0trx.h

```diff
@@ -407,6 +407,9 @@ struct trx_lock_t
 
   /** number of record locks; protected by lock_sys.assert_locked(page_id) */
   ulint n_rec_locks;
+  /** number of lock_rec_set_nth_bit() calls since the start of transaction;
+  protected by lock_sys.is_holder() */
+  ulint set_nth_bit_calls;
```

Comment:
This seems to be protected by `trx_t::mutex` as well. Is that the case? `lock_sys.is_holder()` can also refer to a non-exclusive `lock_sys.latch`.